instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Login.css";
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isLoggedIn: props.isLoggedIn,
+            username: "",
+            password: ""
+        }
+    }
+
+    changeHandler = e => {
+        this.setState({ [e.target.name]: e.target.value })
+    }
+
+    login = () => {
```

Comment:
Possible refactor
When you are doing -> (variable  !== "") you're checking to see if the variable is truthy. Technically this does the same thing -> (variable) because if the variable is true, the code will execute, since an empty string is false. So you could say -> (this.state.username && this.state.password)